src/index.html

```diff
@@ -1,13 +1,369 @@
+            </div>
+            <div class="testimonial testimonials__item grid__item--5-8">
+              <img
+                src="./images/speaker2.jpg"
+                alt="speaker"
+                class="testimonial__photo"
+              >
+              <h3 class="testimonial__name">
+                <a
+                  href="https://www.bose.com/en_us/products/speakers/portable_speakers/soundlink-flex.html#v=soundlink_flex_carmine_red"
+                  class="link"
+                  target="_blank"
+                >
+                  SoundLink Flex Bluetooth speaker
+                </a>
+              </h3>
+              <div class="testimonial__position">Portable bluetooth</div>
+              <div class="testimonial__description">$ 149.00</div>
+            </div>
+            <div class="testimonial testimonials__item grid__item--9-12">
+              <img
+                src="./images/speaker3.jpg"
+                alt="speaker"
+                class="testimonial__photo"
+              >
+              <h3 class="testimonial__name">
+                <a
+                  href="https://www.bose.com/en_us/products/speakers/portable_speakers/soundlink-color-bluetooth-speaker-ii.html"
+                  class="link"
+                  target="_blank"
+                >
+                  SoundLink Color Bluetooth speaker II
+                </a>
+              </h3>
+              <div class="testimonial__position">Portable bluetooth</div>
+              <div class="testimonial__description">$ 129.00</div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section gallery" id="gallery">
+        <div class="container">
+          <h2 class="page__section--title">
+            Browse Bose
+            <br>
+            products by category
+          </h2>
+
+          <div class="location gallery__location">
+            <div class="location__images">
+              <div class="location__image--container grid__item--1-8">
+                <img
+                  src="images/photos/gallery1.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
+              </div>
+              <div class="location__image--container grid__item--9-12">
+                <img
+                  src="images/photos/gallery2.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
+              </div>
+            </div>
+            <div class="location__description">
+              Headphones & earbuds
+            </div>
+          </div>
+
+          <div class="location gallery__location">
+            <div class="location__images">
+              <div class="location__image--container grid__item--1-8">
+                <img
+                  src="images/photos/gallery3.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
```

Comment:
these images should be square in mobile device only 
<img width="702" alt="image" src="https://user-images.githubusercontent.com/100297341/191464761-038c2581-8f21-4c14-8ef7-3703b0d3c2f6.png">